v03_pipeline/lib/reference_datasets/misc.py

```diff
@@ -57,6 +57,12 @@ def filter_mito_contigs(
 
 
 def filter_contigs(ht, reference_genome: ReferenceGenome):
+    if hasattr(ht, 'interval'):
```

Comment:
I made a small change to the logic here.  Instead of skipping this we filter within the interval.  I believe this is the same as what we have in production right now.